src/test/kotlin/zhi/yest/vk/friendfinder/controller/groups/GroupsControllerTest.kt

```diff
@@ -0,0 +1,76 @@
+package zhi.yest.vk.friendfinder.controller.groups
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.given
+import kotlinx.coroutines.runBlocking
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.eq
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.http.MediaType.APPLICATION_JSON_UTF8
+import org.springframework.test.context.junit4.SpringRunner
+import org.springframework.test.web.reactive.server.WebTestClient
+import zhi.yest.vk.friendfinder.config.security.dto.VkError
+import zhi.yest.vk.friendfinder.config.security.dto.VkException
+import zhi.yest.vk.friendfinder.domain.Group
+import zhi.yest.vk.friendfinder.vk.GroupsService
+
+@RunWith(SpringRunner::class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class GroupsControllerTest {
```

Comment:
Cannot run the tests - Idea and Gradle don't see it :(

Reply:
Spek looks good for unit tests, but for integration tests with spring context it can hardly work well. I found this thread - https://github.com/spekframework/spek/issues/50, seems like there was an extension, but they passed to do it - https://github.com/spekframework/spek-spring-extension. Please let me know if I'm wrong and it's possible to write specs for Spring Boot Tests